keps/693-multikueue/README.md

```diff
@@ -316,6 +323,76 @@ that ware created by this multikueue manager cluster and delete them if their lo
 - `WorkerLostTimeout` - defines the time a local workload's multikueue admission check state is kept Ready
 if the connection with its reserving worker cluster is lost.
 
+
+### MultiKueue Dispatcher API
+
+The MultiKueue Workload Controller consists of nomination nd synchronization: 
+first, it nominates worker clusters for each workload based on the configured dispatching mode;
+then, it synchronizes the workload by creating and managing its copies in the nominated clusters. 
+This ensures that only one cluster ultimately admits and runs the workload, 
+while the management cluster maintains up-to-date status by tracking the admitted job.
+
+#### Cluster Nomination
+Cluster nomination phase looks differently depending on the configured dispatching mode.
+MultiKueue supports three dispatching modes: AllAtOnce (`kueue.x-k8s.io/multikueue-dispatcher-all-at-once`),
+Incremental (`kueue.x-k8s.io/multikueue-dispatcher-incremental`), and External.
```

Comment:
In order to meet the requirements of [Story 3](#story-3) we introduce an API for custom dispatching algorithms.
We also, implement a built-in incremental dispatcher as a reference implementation. 

Including the pre-existing dispatching algorithm (using all at once approach) we distinguish the following dispatchers: